examples/multiple-apps/.gitignore

```diff
@@ -0,0 +1 @@
+build
```

Comment:
Probably it makes more sense to exclude the `build` folder altogether, unless you plan to have anything non-generated in that folder.